### DISCUSSION TITLE
Photoshop inputs for mask? / More Efficient Masking Methods?

### DISCUSSION THREAD

**Discussion by heartlocket** [hlky/stable-diffusion-webui#751] [General]
Hi there, 

The mask feature really does not work in an efficient way. I was wondering, is it possible to just mask the input image with a brush in photoshop and then upload? If so, what would the correct color etc settings be for it to recognize the masked image?

**Comment by A2va** [comment]
A PR has already been merged #636

**Comment by Heathen** [comment]
"what would the correct color etc settings be for it to recognize the masked image?"

I would like a reply to this. I've tried using transparency, but that didn't work as a mask.

**Reply by A2va** [reply]
Try to put black instead of transparency. The mask is supposed to be in RGB and not RGBA

**Reply by codedealer** [reply]
There is no code currently to handle masks uploaded from 3rd party software. Mask indeed expects RGB but only when passed from Gradio.
